src/components/server/hooks/useLoadServer.ts

```diff
@@ -0,0 +1,47 @@
+import { Repo } from "@automerge/automerge-repo";
+import { YtServer } from "../../serversContext/types";
+import { useMemo } from "react";
+import { useServerName } from "../../../lib/helpers/useServerName";
+import {
+  serversHandler,
+  useServersContext,
+} from "../../serversContext/serversContext";
+
+type LoadServerStatus =
```

Comment:
We could export a `ServerStatusEnum` instead of using strings (e.g in `Server.tsx`)